rust/examples/decompile/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
```

Comment:
Unless you're using this, please remove it; We only need it for examples we're building in CI.  We could definitely beef up our Rust API testing, but that should probably have a focused effort later on.

Reply:
Ah, I just copied this over from the basic example

Reply:
addressed in 12afe637b60855ccba51d5dfe81b6bfece859534 -- is this enough?

Reply:
I meant the whole file